assembl/static2/js/app/components/debate/multiColumns/postColumn.jsx

```diff
@@ -43,7 +42,7 @@ const PostColumn = ({
   width,
   withColumnHeader
 }: Props) => (
-  <div className="column-view" style={{ width: width }}>
+  <div className="column-view" style={width && { width: width }}>
```

Comment:
do you really need this ? From what I remember, the property is ignored if its value is `undefined`

Reply:
You're right, this is not needed.